components/crud-web-apps/volumes/backend/apps/rok/routes/delete.py

```diff
@@ -0,0 +1,31 @@
+from werkzeug import exceptions
+
+from kubeflow.kubeflow.crud_backend import api, logging
+
+from ...common import utils as common_utils
+from .. import utils as rok_utils
+from . import bp
+
+log = logging.getLogger(__name__)
+
+
+@bp.route("/api/namespaces/<namespace>/pvcs/<pvc>", methods=["DELETE"])
+def delete_pvc(pvc, namespace):
+    """
+    Delete a PVC, even if it is only mounted on PVCViewer Pods.
```

Comment:
It seems like the Dockerfile for the PVCViewer is missing in this PR. @kimwnasptd can you confirm this?

Reply:
Correct. This feature is a part of Rok and that's why it's only present in the Rok flavor.
We are evaluating how this specific feature could be integrated in the vanilla Kubelfow and not rely on Rok functionalities and code

Reply:
@kimwnasptd I think I already have a solution for this, however, a controller will be needed for this as well. I already have a working container image for the file browsing, drag/drop, downloading, zip/unzip, etc.

Reply:
Most probably yes, but there are a lot of details to discuss about this. Lets have this idea in our backlog and discuss it more and see what would make sense for us to implement after 1.3